BlogTemplate/Pages/Index.cshtml

```diff
@@ -1,15 +1,22 @@
 @page
 @model IndexModel
-@{
-    ViewData["Title"] = "Home Page";
-
-    foreach (var post in Model.PostSummaries)
-    {
+@{
+    ViewData["Title"] = "Home Page";
+
+    @if (!Model.PostSummaries.Any())

```

Comment:
I don't think you need the @ on this line since you're already in a code block (`@{}`)